content/docs/user-guide/project-structure/dvcyaml-files.md

```diff
@@ -625,17 +626,22 @@ Refer to [Visualizing Plots] and `dvc plots show` for examples.
 - `x` (string) - column/field name from which the X axis data comes from. An
   auto-generated _step_ field is used by default.
 
+  - Top-level plots: Accepts string or dictionary. For dictionaries, the keys
```

Comment:
it's quite hard to unpack this tbh ... do brief snippets right here? Even after reading it a few times, I can't understand all the options :)